zuul.d/kuttl_multinode.yaml

```diff
@@ -71,3 +71,15 @@
       commands_after_kuttl_run:
         - oc get pv
         - oc get all
+
+- job:
+    name: cifmw-multinode-kuttl-operator-target
+    parent: cifmw-base-multinode-kuttl
```

Comment:
there isnte really a need to make it multi node.

in the nova one i override the noeset to be single node 

https://github.com/openstack-k8s-operators/nova-operator/blob/main/.zuul.yaml#L41
the parent is a multi node job but there is no benefit to multi node here

i would suggest adding 

 nodeset: centos-9-medium-crc-extracted-2-39-0-3xl 

and updating the name.